pkg/local_workflows/code_workflow.go

```diff
@@ -0,0 +1,46 @@
+package localworkflows
+
+import (
+	"os"
+
+	"github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/workflow"
+	"github.com/spf13/pflag"
+)
+
+const (
+	codeWorkflowName = "code.test"
+)
+
+// WORKFLOWID_CODE defines a new workflow identifier
+var WORKFLOWID_CODE workflow.Identifier = workflow.NewWorkflowIdentifier(codeWorkflowName)
+
+// InitCodeWorkflow initializes the code workflow before registering it with the engine.
+func InitCodeWorkflow(engine workflow.Engine) error {
+	// register workflow with engine
+	flags := pflag.FlagSet{}
+	_, err := engine.Register(WORKFLOWID_CODE, workflow.ConfigurationOptionsFromFlagset(&flags), codeWorkflowEntryPoint)
+	return err
+}
+
+// codeWorkflowEntryPoint is the entry point for the code workflow.
+// it provides a wrapper for the legacycli workflow
+func codeWorkflowEntryPoint(invocationCtx workflow.InvocationContext, _ []workflow.Data) (output []workflow.Data, err error) {
+	// get necessary objects from invocation context
+	config := invocationCtx.GetConfiguration()
+	logger := invocationCtx.GetLogger()
```

Comment:
Suggestion: There is an [enhanced logger](https://github.com/snyk/go-application-framework/blob/main/pkg/workflow/invocationcontextimpl.go#L79), that allows level logging. Not necessary now but generally a good approach to use this logger.

Reply:
https://github.com/snyk/go-application-framework/pull/131/commits/14996bd1b700a47ad756a0cfabece8968b5efbb4